package-lock.json

```
@@ -1,5 +1,5 @@
 {
-    "name": "contao",
+    "name": "contao-contao",
```

```suggestion
    "name": "contao",
```

Just reverting to the initial state